testing/endpoint.go

```diff
@@ -165,6 +165,18 @@ func (endpoint *Endpoint) UpdateClient() (err error) {
 	return endpoint.Chain.sendMsgs(msg)
 }
 
+// ProvideCounterparty will construct and execute a MsgProvideCounterparty on the associated endpoint.
+func (endpoint *Endpoint) ProvideCounterparty() (err error) {
+	merklePath := commitmenttypes.NewMerklePath([]byte("ibc"), []byte(""))
```

Comment:
`Endpoint` should probably be modified to have the merkle prefix now

Reply:
Agreed, let's create an issue and do this later.

Think it should be added as a new field but this should be the default value that can then be malleated later by test writer